### PR TITLE
Added run as root check

### PR DESCRIPTION
**Pull request by sunmorgus**
Added a check to ensure the script is running as root (tripped me up the first time I ran it, lol).

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ set -e
 
 apt-get update
 apt-get upgrade -y
-apt-get install -y bluez-tools pulseaudio-module-bluetooth pulseaudio
+apt-get install -y --force-yes bluez-tools pulseaudio-module-bluetooth pulseaudio
```
Comment (hadess) [inline setup.sh]: Can you please explain what this is for in the commit message?

File: setup.sh [hunk 2]
```diff
@@ -21,9 +21,9 @@ set -e
 
 # Update, install the necessary plugins
 
-apt-get update
-apt-get upgrade -y
-apt-get install -y --force-yes bluez-tools pulseaudio-module-bluetooth pulseaudio
+# apt-get update
+# apt-get upgrade -y
+# apt-get install -y --force-yes bluez-tools pulseaudio-module-bluetooth pulseaudio
```
Comment (hadess) [inline setup.sh]: This is obviously wrong.

File: setup.sh [hunk 3]
```diff
@@ -21,9 +21,12 @@ set -e
 
 # Update, install the necessary plugins
 
-# apt-get update
+apt-get update
+
+# Running upgrade has consistently killed my chip
```
Comment (hadess) [inline setup.sh]: Then it's something to be debugged.

**Comment by hadess** [comment]
Nothing I'd take here, apart from the first commit. Do you want to update your PR and split off the other problems?

**Comment by sunmorgus** [comment]
I'm sorry, I had no idea that those commits would make it into the pull request too. Other than the very first one, they were just things that I was doing for my own needs. I'll update the PR :)

**Comment by hadess** [comment]
You need to reset the branch to the commit that includes the one change you want here:
`git reset --hard <commit id>`
And then force push:
`git push -f`

You'll lose your local changes, but they shouldn't be too hard to re-add later on. There should be only one commit appearing in this merge request, and on top of the changes originally in this branch.

**Comment by sunmorgus** [comment]
Done. Sorry about that, still new to pull requests.

**Pull request merged**